.github/workflows/package.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           sudo apt update && sudo apt -y install make pkg-config rubygems && sudo gem install fpm
 
           # Download corazawaf/coraza/coraza.conf for distribution
-          curl https://raw.githubusercontent.com/corazawaf/coraza/v2/master/coraza.conf-recommended > coraza.conf
+          curl https://raw.githubusercontent.com/corazawaf/coraza/v3.0.2/coraza.conf-recommended > coraza.conf
```

Comment:
Currently we are using [3.0.0](https://github.com/corazawaf/coraza-spoa/blob/dd5eb86d1e9abbdd5fe568249f36a6d85257eba7/go.mod#L7). So, let's make version bump synchronous in another PR.